documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -825,7 +860,7 @@ validator factory still is referenced by application code.
 ====
 
 [[section-time-provider]]
-=== Time providers for `@Future` and `@Past`
+=== Time providers for @Future and @Past
```

Comment:
I don't mind, but why did you remove it? Does it look bad with the mark-up in titles?